FlowCrypt/Functionality/Services/WKDURLsApi.swift

```diff
@@ -37,9 +40,10 @@ class WKDURLsApi: WKDURLsApiType {
             }
             let pubKeys = safeResponse.keyDetails
                     .filter { !$0.users.filter { $0.contains(email) }.isEmpty }
-                    .map(\.public)
             resolve(pubKeys)
         }
+        .timeout(Constants.lookupEmailRequestTimeout)
+        .recoverFromTimeOut(result: [])
```

Comment:
This is correct for WKD - because the recipient may or may not have a WKD server set up.

However, timeout is not the only failure mode here. If their server is not reachable, that should also result in `[]` (when servers are not set up, sometimes it results in a timeout and sometimes it results in "server not reachable" error right away - need to handle both).

Reply:
it will return [] if any error occurs with request.

Reply:
Yes and no. 
 - If any error occurred when getting the `/policy` file, then that's a `[]`
 - However, if you are already past the policy stage, and now querying the URL of the public key, then only `HTTP 404` or timeout results in `[]`. Other errors should be escalated as usual.

I'll make an another issue for this.